### PR TITLE
Throws, explicits and trivial cleanups

### PR DESCRIPTION
**Pull request by miguelportilla**
Reviewers: @seelabs @nbougalis

### CODE REVIEW

File: src/ripple/ledger/impl/View.cpp [hunk 1]
```diff
@@ -1355,7 +1355,7 @@ rippleSend (ApplyView& view,
     {
         // VFALCO Why do we need this bCheckIssuer?
         // Direct send: redeeming IOUs and/or sending own IOUs.
-        terResult   = rippleCredit (view, uSenderID, uReceiverID, saAmount, false, j);
+        rippleCredit (view, uSenderID, uReceiverID, saAmount, false, j);
```
Comment (seelabs) [inline src/ripple/ledger/impl/View.cpp]: We don't have to check the return value of `rippleCredit`? That doesn't look right.

Comment (nbougalis) [inline src/ripple/ledger/impl/View.cpp]: We do not: notice what happens two lines down: `terResult = tesSUCCESS`. There's a reason for this (even if it's a bit esoteric): the only errors that `rippleCredit` can return come from `trustCreate` and, obviously, there's no trust line that needs to be created if the sender or the receiver are the issuer (it must exist) or if XRP is being sent.

**Comment by seelabs** [comment]
:+1:

**Comment by nbougalis** [comment]
:+1:

File: src/ripple/ledger/impl/View.cpp [hunk 2]
```diff
@@ -1355,7 +1355,7 @@ rippleSend (ApplyView& view,
     {
         // VFALCO Why do we need this bCheckIssuer?
```
Comment (nbougalis) [inline src/ripple/ledger/impl/View.cpp]: Please remove this obsolete comment.

**Comment by nbougalis** [comment]
Merged as ed9f5639a8e02d1931dd45f13fa2fa21827f1a4d

**Pull request closed without merging**